.ci/scripts/retry.sh

```
@@ -1,3 +1,4 @@
+#!/bin/bash -e
```

retry was suppressing errors